_posts/2022-09-06-deepchecks-and-zenml.md

```diff
@@ -507,4 +507,4 @@ or the tutorial featured in this article, we encourage you to join our
 [weekly community hour](https://www.eventbrite.com/e/zenml-meet-the-community-tickets-354426688767).
 
 If you want to know more about ZenML or see more examples, check out our
-[docs](https://docs.zenml.io) and [examples](https://github.com/zenml-io/zenml/tree/main/examples).
+[docs](https://docs.zenml.io/v/0.13.2) and [examples](https://github.com/zenml-io/zenml/tree/main/examples).
```

Comment:
I don't get this change, why links to 0.13.2 docs here?

Reply:
Same for two other posts below where 0.20.0 / 0.21.0 are hardcoded in several places

Reply:
I have written an explanation about this in the main thread.